common/app/routes/challenges/redux/bug-saga.js

```diff
@@ -12,13 +10,11 @@ function filesToMarkdown(files = {}) {
     }
     const fileName = moreThenOneFile ? `\\ file: ${file.contents}` : '';
     const fileType = file.ext;
-    return fileString + dedent`
-      \`\`\`${fileType}
-      ${fileName}
-      ${file.contents}
-      \`\`\`
-      \n
-    `;
+    return fileString +
+      '\`\`\`' + fileType + '\n' +
```

Comment:
@BerkeleyTrue 👍 I was trying to make it look like it did with dedent, but I'm OK with that as well. I updated the PR.